### PR TITLE
fix: update the return type and remove unnecessary branches/tests in `blas/ext/base/ssumpw`

### PR DESCRIPTION
**Pull request by headlessNode**
## Description

> What is the purpose of this pull request?

This pull request:

-   update the return type of the `blas/ext/base/ssumpw`
-   remove redundant branches and related test cases

## Related Issues

> Does this pull request have any related issues?

No.

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

**Review by kgryte — APPROVED** [review]
LGTM. Thanks, @headlessNode!

File: lib/node_modules/@stdlib/blas/ext/base/ssumpw/src/main.c [hunk 1]
```diff
@@ -37,9 +37,9 @@
 * @param strideX  stride length
```
Comment (kgryte) [inline lib/node_modules/@stdlib/blas/ext/base/ssumpw/src/main.c:36]: @headlessNode Don't we also need to remove the `is_nanf` dependency both here and in the `manifest.json` file?

Ref: https://github.com/stdlib-js/stdlib/commit/c83f10824d14d8c0290b68a5f5bcedbdc2c52c6d#r149865541

**Review by kgryte — COMMENTED** [review]
@headlessNode Left one last question. Once resolved, this should be ready for merge.

**Comment by headlessNode** [comment]
@kgryte done.

**Review by kgryte — APPROVED** [review]
LGTM. Thanks, @headlessNode!

**Pull request merged**